### ISSUE TITLE
javafx support

### ISSUE THREAD

**Issue by triathematician** [triathematician/blaise-app#4]

**Comment by cogniveon** [comment]
What kind? Do you want to port the app to javafx? Or just use this
https://docs.oracle.com/javafx/8/embed_swing/jfxpub-embed_swing.htm

This is my first open source contribution..but I'm confident with JavaFX

**Comment by triathematician** [comment]
Yes to porting... basically looking for something that replicates the "menu config" system implemented here for swing but for javafx -- plus the equivalent of being able to link the menu action directly to code with an "@Action" annotation.
